app/controllers/application_controller.rb

```diff
@@ -74,9 +75,10 @@ def metrics_logger
   end
 
   def ensure_visit_integrity
-    unless visit && visit.prisoner && visit.prisoner.prison_name.present?
+    unless visit && visit.prisoner && visit.prison_name.present?
       redirect_to edit_prisoner_details_path,
-        notice: 'You need to complete missing information to start or continue your visit request'
+        notice: 'You need to complete missing information to start' \
```

Comment:
@dan-palmer @SteveMarshall @threedaymonk @tatyree I just saw that I forgot a space at the end, but tests didn't fail. I believe we want that space right?